app/views/pages/home.html.haml

```diff
@@ -31,10 +31,15 @@
       .container
         .column18.line
           %h2.slack.icon
-        .column5
-        .column9
-          %p B'more on Rails members regularly hang out in Slack, enter your email below to get an invite and come say "hi"!
-          = form_tag slack_invite_url do
+        .column3
+        .column12
+          %p
+            B'more on Rails members regularly hang out in Slack.
+            %br
+            Send us an email at <a href='mailto:bmoreonrails@gmail.com?subject=Slack Invite Request'>bmoreonrails@gmail.com</a>
+            %br
+            to receive an invite and come say "hi"!
+          -#= form_tag slack_invite_url do
```

Comment:
Should we go ahead and delete this whole form instead of commenting it out? If we ever want to bring it back (if Slack ends up opening up that API) we can find it in git.